inputs/US/manual_input_usa.csv

```diff
@@ -58,65 +58,21 @@ Coal integrated retrofit 90%-CCS,efficiency,2030,0.386,p.u.,-,"NREL, “Cost and
-Natural gas steam reforming,FOM,2030,0.05,%/year,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Coal gasification,FOM,2020,0.06,%/year,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Coal gasification,FOM,2030,0.06,%/year,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Heavy oil partial oxidation,FOM,2020,0.05,%/year,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Solid biomass steam reforming,efficiency,2020,0.712,p.u.,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Biomass gasification,efficiency,2020,0.35,p.u.,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Biomass gasification,efficiency,2050,0.525,p.u.,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Natural gas steam reforming CC,efficiency,2030,0.637,p.u.,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Natural gas steam reforming CC,efficiency,2050,0.695,p.u.,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Coal gasification CC,efficiency,2030,0.532,p.u.,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Coal gasification CC,efficiency,2030,0.609,p.u.,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Biomass gasification CC,efficiency,2030,0.328,p.u.,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Biomass gasification CC,efficiency,2040,0.514,p.u.,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Natural gas steam reforming,investment,2020,248.58,USD/kW,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Natural gas steam reforming,investment,2030,210.55,USD/kW,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Coal gasification,investment,2020,565.81,USD/kW,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Coal gasification,investment,2030,466.83,USD/kW,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Heavy oil partial oxidation,investment,2020,574.2,USD/kW,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
```

Comment:
Not all of the once that you removed here made it into the generic `manual_input.csv`, is that intentional?